crates/ethereum-forks/src/lib.rs

```diff
@@ -15,6 +15,10 @@
 // TODO: remove when https://github.com/proptest-rs/proptest/pull/427 is merged
 #![allow(unknown_lints, non_local_definitions)]
 #![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(not(feature = "std"))]
+extern crate alloc;
```

Comment:
why not do this in the manifest?

Reply:
I have used this as a reference from earlier PRs like https://github.com/paradigmxyz/reth/pull/8729 and https://github.com/paradigmxyz/reth/pull/8817.